### PR TITLE
Add the versioning number

### PR DESCRIPTION
**Pull request by RemiLehe**
This is done in a similar way as https://jeffknupp.com/blog/2013/08/16/open-sourcing-a-python-project-the-right-way/

### CODE REVIEW

**Comment by ax3l** [comment]
haha, I know that blog post, it's a good one! :sparkles:

File: setup.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
+import opmd_viewer  # In order to extract the version number
```
Comment (ax3l) [inline setup.py]: this is part I am not 100% sure in the original blog post with:

the include has quite some dependencies. we could also create a single version.py file that we both include in `setup.py` and in ` o[pen]pmd_viewer/__init__.py`. By that we could depend on a more lightweight include with fewer side effects and still have the version number in one place for both the packaging and the module.

**Comment by RemiLehe** [comment]
Ok, how about adding the version in `opmd_viewer/version.py` instead of `opmd_viewer/version/version.py`? Any preference?

**Comment by RemiLehe** [comment]
Actually, now that I think about it, I am not sure that the solution you proposed will work: as I understand it, your solution is to replace, in `setup.py`
```
from opmd_viewer import __version__
```
by
```
from opmd_viewer.version.version import __version___
```
However, when executing the above import statement, Python will start by executing the `opmd_viewer.__init__.py`, and thus it will still import all the packages that are associated with opmd_viewer.

Am I correct?

**Comment by ax3l** [comment]
What I thought is that python will only execute `opmd_viewer.version.__init__.py` but not the overlaying `opmd_viewer.__init__.py` but you might be right. In that case just leave it as it is :)

**Comment by RemiLehe** [comment]
I did a few tests and it confirms that the `opmd_viewer.__init__.py` get executed, even if you are only importing and sub-sub-module.

Should the current PR be merged then?

**Comment by ax3l** [comment]
hm, then we leave it as it is I guess. thank you a lot for checking!

File: setup.py [hunk 2]
```diff
@@ -9,7 +10,7 @@
 
 # Main setup command
 setup(name='opmd_viewer',
-      version='1.0',
```
Comment (ax3l) [inline setup.py]: hopefully this does not cause problems if someone has already installed the current version and then goes to install the first release aka `0.1.0`

**Pull request merged**